### DISCUSSION TITLE
HashTag to provide exact match of articles

### DISCUSSION THREAD

**Discussion by shamnad-sherief** [dailydotdev/daily#800] [Feature Requests]
There should be a tagging system where i can see articles related to the tag. The problem now i face is, when i search for Java it shows articles related to Javascript. With a tag management system in place, I would be able to see articles that are specifically related to the tag that I'm searching for, making it easier to find the information I need.

**Comment by rebelchris** [comment]
Hi @shamnad-sherief 

We do have a tag system in place. Have you tried setting up your customized feed?
You can find the instructions here: https://docs.daily.dev/docs/setting-up-your-feed/filtering-content-feed

Alternatively, you can also use this screen to view only posts for a specific tag, which might be what you are looking for.
See the screenshot below.
<img width="1051" alt="Screenshot 2023-01-20 at 08 58 44" src="https://user-images.githubusercontent.com/554874/213645537-96804333-a694-43f9-86a7-eeb1f04ee7ee.png">

**Reply by shamnad-sherief** [reply]
![image](https://user-images.githubusercontent.com/66134967/213647980-9789f7ba-4687-4208-9be0-e195c3e33bd0.png)

I am discussing about the search bar. It is good if you guys introduce something to pickup the exact match when someone search.
